modules/cbr-mutli-rule-module/README.md

```diff
@@ -0,0 +1,105 @@
+# Multi CBR Rule Module
+
+Creates multiple rules for Context Based Restrictions
+
+## Usage
+
+```hcl
+module "cbr_rules" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-rule-module"
+  version          = "1.9.0"
```

Comment:
should we update to:
`version = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release`